src/extension/background-script/connectors/alby.ts

```diff
@@ -122,17 +140,22 @@ export default class Alby implements Connector {
     const cacheValue = this._cache.get(cacheKey) as GetInfoResponse<
       WebLNNode & GetAccountInformationResponse
     >;
-    if (cacheValue) {
+
+    const limit = await this.getLimit();
+
+    if (cacheValue && cacheValue.data.limit === limit) {
       return cacheValue;
     }
 
     try {
       const info = await this._request((client) =>
         client.accountInformation({})
       );
```

Comment:
can we use the new endpoint instead of this one? it should contain the same information and the additional account limit info. 

so we don't need to do a second request.